lib/service/Controller/Swap/SwapController.dart

```diff
@@ -53,7 +53,7 @@ class SwapController extends GetxController {
         ERC20(address: tokenAAddress, client: controller.client.value);
     //ERC20 tokenB =
     //ERC20(address: tokenBAddress, client: controller.client.value);
-    BigInt rate = BigInt.parse("100000000000000000000000");
+    BigInt rate = BigInt.parse("1000000000000000000000000");
```

Comment:
Add a comment or name this big number as a constant to give it more meaning.

`const final int QUINTILLION = 1000000000000000000`;